msal-python-conceptual/index.md

```diff
@@ -31,22 +29,36 @@ Install the MSAL for Python package. You can find MSAL Python on [PyPI](https://
 pip install msal
 ```
 
-## Setting up
+## Identity concepts
+
+MSAL Python is part of the [Microsoft identity platform](/entra/identity-platform/v2-overview) ecosystem. It's important to farmiliarize yourself with the following concepts to effectively use MSAL Python to protect your applications and APIs:
+
+- [Identity and access management](/entra/fundamentals/identity-fundamental-concepts)
+- [Authentication and authorization](/entra/identity-platform/authentication-vs-authorization)
+- [OAuth 2.0 and OpenID Connect (OIDC) in the Microsoft identity platform](/entra/identity-platform/v2-protocols)
+- [Confidential and public client accounts in the Microsoft identity platform](/entra/identity-platform/msal-client-applications)
+- [Security tokens](/entra/identity-platform/security-tokens)
+
+## Usage scenarios
 
-Before using MSAL Python, make sure to [register your application](/azure/active-directory/develop/quickstart-v2-register-an-app) with the Microsoft identity platform. You will need to take note of your **client ID** and **tenant ID**.
+To use MSAL Python, register an application with the Microsoft identity platform. You'll need an Azure account with an active subscription. [Create a free account](https://signup.azure.com/) if you don't have one. You can register your app in a [customer tenant](/entra/external-id/customers/quickstart-tenant-setup) or [workforce tenant](/entra/identity-platform/scenario-web-app-sign-user-app-registration?tabs=python).
 
->[!IMPORTANT]
->When registering the application, make sure that you set up **redirect URLs** within the **Authentication** blade. Redirect URLs vary depending on the target platform.
->
->![Screenshot showing redirect URLs in Azure Portal](./media/redirect-urls.png)
->
->For desktop and mobile applications, make sure you add `http://localhost` as redirect URL if you do not rely on authentication brokers.
+MSAL Python can be used by applications to acquire tokens to access protected APIs. Tokens can be acquired by different application types. These app types include desktop applications, web applications, web APIs, and applications running on devices that don't have a browser (such as IoT devices). Different app types follow different auth flows.
```

Comment:
```suggestion
Applications can use MSAL Python to acquire tokens for accessing protected APIs. Different app types acquire tokens using different auth flows. The supported app types include desktop applications, web applications, web APIs, and applications running on devices that don't have a browser (such as IoT devices). 
```

Reply:
Not sure if this bit sounds any better, but the idea is to eliminate the passive voice in two consecutive sentences.